google-cloud-contrib/google-cloud-logging-adapters/cloud-logging-over-slf4j/src/main/java/org/slf4j/impl/CloudLoggingAdapter.java

```diff
@@ -0,0 +1,312 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.slf4j.impl;
+
+import com.google.cloud.logging.Severity;
+import java.io.Serializable;
+import org.slf4j.Logger;
+import org.slf4j.Marker;
+import org.slf4j.helpers.FormattingTuple;
+import org.slf4j.helpers.MessageFormatter;
+
+public class CloudLoggingAdapter implements Logger, Serializable {
+
+  private static final long serialVersionUID = 1L;
+  private CloudLogger cloudLogger;
+
+  CloudLoggingAdapter(String loggerName) {
+    try {
+      cloudLogger = new CloudLogger(loggerName);
+    } catch (Exception e) {
+      e.printStackTrace();
+    }
```

Comment:
Why are you catching and hiding the exception? You will leave the class in an unusable state, with `cloudLogger` set to null.